lib/signet/oauth_2/client.rb

```diff
@@ -109,6 +111,7 @@ def initialize options = {}
         @state                = nil
         @username             = nil
         @access_type          = nil
+        @granted_scopes       = nil
```

Comment:
you can call it `scopes`.

Reply:
We already have a field called `scope` which represents request scopes. So I thought it'd be better to keep it explicit here. WDYT?

Reply:
Makes sense